packages/remix-netlify-edge/server.ts

```diff
@@ -0,0 +1,70 @@
+import type { ServerBuild } from "@remix-run/server-runtime";
+import { createRequestHandler as createRemixRequestHandler } from "@remix-run/server-runtime";
```

Comment:
They should indeed 👍

Reply:
Looks like this is not possible because when it compiles it complains about the *.ts extensions that are importing modules within the `@remix-run/deno` module and deno requires file extensions for imports.

> Deno supports both JavaScript and TypeScript as first class languages at runtime. This means it requires fully qualified module names, including the extension

https://deno.land/manual@v1.1.0/getting_started/typescript#using-typescript

```
packages/remix-deno/index.ts:2:42 - error TS2691: An import path cannot end with a '.ts' extension. Consider importing './sessions/fileStorage.js' instead.

2 export { createFileSessionStorage } from "./sessions/fileStorage.ts";
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~

...
```

Is it OK to stay with `@remix-run/server-runtime` here? Also, I haven't run the Netlify Edge template yet but I imagine I'll run into the same issue there potentially, mind you, at that point it's referencing a bundled package.

Reply:
The problem is because as this stands, Remix is trying to bundle the Deno runtime files and as the message says, the import styles are incompatible. As you predict, this will affect all sites that use the adapter. The inlined version of the Deno package that we used in the previous version of the adapter changed the imports to be node-style. By the time Deno sees them, they're bundled anyway. I'm surprised this doesn't affect Deno sites that use the new Deno runtime. Are they not being bundled?

Reply:
Just tagging folks for visibility here as this is currently a blocker @MichaelDeBoey @pcattori.

Reply:
You might be able to unblock it by switching back to `server-runtime`

Reply:
Looking at it, we don't actually want the Deno one, because Netlify Edge Functions handle requests differently than regular Deno. We _do_ need the version from `server-runtime`. Even when we inlined the Deno runtime we still used the version from `server-runtime` rather than the Deno one.

Reply:
That sounds like Netlify Edge should change how it handles these requests then. 🤔
Maybe the public API is different (that's why we have these adapters), but at the end of the day Netlify Edge sends them over a Deno environment, so it should work I guess?

I don't have much Deno experience, so @pcattori could probably shed a better light on this.

Reply:
Hey folks! I didn't forget about this PR. I was stuck on the above-mentioned issue in this thread, there was the Remix Conf and I got pulled into something else. I'll be resuming this work next week.

Reply:
@nickytonline Take your time, you don't have any obligations 😉

Reply:
What is actually causing the `import path cannot end with a '.ts' extension`? Is `npm run build` (ie. `remix build`) causing that? or is that error happening when the built bundles are run via `npm run start`?